boards/same54-xpro/include/board.h

```diff
@@ -75,6 +75,14 @@ extern "C" {
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name ATA6561 STANDBY pin definition
+ * @{
+ */
+#define AT6561_STBY_PIN            GPIO_PIN(PC, 13)
+#define AT6561_STBY_MODE           GPIO_OUT
```

Comment:
this define is not needed, if there is a standby pin it is an out_pin from the mcu pov

Reply:
i was just referring to the MODE